Cargo.toml

```diff
@@ -29,4 +29,4 @@ sha256 = "1.5.0"
 syn = "1.0"
 tempfile = "3.3.0"
 wasmparser = "0.119.0"
-pocket-ic = "3.1.0"
+pocket-ic = "4.0.0"
```

Comment:
The title says "bump to v5.0.0" but here we bump to 4.0.0. Which one do we want after all?

Reply:
This is the library which we indeed want to bump to v4.0.0.